### ISSUE TITLE
Issue running command [gcloud preview app deploy].

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#577]
```
Issue running command [gcloud preview app deploy].

What steps will reproduce the problem?


What is the expected output? What do you see instead?


Please provide any additional information below.


Traceback (most recent call last):
  File "googlecloudsdk/gcloud_main.py", line 169, in main
    gcloud_cli.Execute()
  File "googlecloudsdk/calliope/cli.py", line 646, in Execute
    result = args.cmd_func(cli=self, args=args)
  File "googlecloudsdk/calliope/backend.py", line 1394, in Run
    resources = command_instance.Run(args)
  File "surface/preview/app/deploy.py", line 501, in Run
    images.get(module))
  File "googlecloudsdk/api_lib/app/appengine_api_client.py", line 90, in DeployModule
    return operations.WaitForOperation(self.client.apps_operations, operation)
  File "googlecloudsdk/api_lib/app/api/operations.py", line 69, in WaitForOperation
    encoding.MessageToPyValue(completed_operation.error)))
  File "googlecloudsdk/api_lib/app/api/requests.py", line 69, in ExtractErrorMessage
    message=error_details.get('message', '')))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xd7' in position 
118: ordinal not in range(128)

Installation information:

Google Cloud SDK [97.0.0]

Platform: [Mac OS X, x86_64]
Python Version: [2.7.6 (default, Jan  7 2014, 18:34:52)  [GCC 4.2.1 Compatible 
Apple LLVM 5.0 (clang-500.2.79)]]
Python Location: 
[/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/Resourc
es/Python.app/Contents/MacOS/Python]
Site Packages: [Disabled]

[output truncated]

```

Original issue reported on code.google.com by `rui.y...@ascend.io` on 23 Feb 2016 at 11:57

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this. We did a big cleanup of a lot of handling of 
non-ASCII messages recently, but appear to have missed this one.

You can run again with --log-http and check the logs for the error message in 
question.
```

Original comment by `z...@google.com` on 24 Feb 2016 at 3:35
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue should be fixed in Cloud SDK release 99.0.0. Please comment if 
you're still experiencing this issue.
```

Original comment by `z...@google.com` on 8 Mar 2016 at 4:41
- Changed state: **Fixed**
